subtests/docker_cli/systemd/systemd.py

```diff
@@ -0,0 +1,71 @@
+"""
+Summary
+-------
+
+Operational Summary
+---------------------
+    1. Copy Unit file
+    2. Build image using predefined Dockerfile & p4321-server.py
+    3. Run image
+    4. Confirm p4321-server script is running
+    5. Stop container
+    6. cleanup
+
+"""
+
+import shutil
+import time
+from os.path import join
+from autotest.client import utils
+from autotest.client.shared.utils import is_port_free
+from dockertest.subtest import Subtest
+from dockertest.dockercmd import DockerCmd
+
+
+class systemd(Subtest):
+
+    def initialize(self):
+        super(systemd, self).initialize()
+        sysd_unitf = join(self.bindir, 'p4321.service')
+        shutil.copyfile(sysd_unitf, self.config['sysd_unitf_dest'])
+        for build_file in ['Dockerfile', 'p4321-server.py']:
+            shutil.copy(join(self.bindir, build_file), self.tmpdir)
+        self.build_image()
+
+    def build_image(self):
+        dkrcmd = DockerCmd(self, 'build', ['--force-rm -t',
+                                           self.config['name'], self.tmpdir])
+        dkrcmd.execute()
```

Comment:
What's your thought behind building from ``tmpdir``.  Why not skip the copy entirely and
```python
       dkrcmd = DockerCmd(self, 'build', ['--force-rm -t',
                                          self.config['name'], self.bindir])
```

?